ml/pre-processing/PreProcessingConfig.scala

```diff
@@ -1,9 +1,9 @@
-object PreProcessingConfig {
-
-  val keyspacekeyspace = "gsoc3"
+object PreProcessingConfig extends Serializable{
```

Comment:
Okay, I use the library: pyhocon to deal with the JSON format configuration file for python.
(Actually, the configuration file is in the Hocon format, but it has the same file format with JSON)
I also combine the two configuration files into one.